packages/sftpgo/build.sh

```diff
@@ -0,0 +1,48 @@
+TERMUX_PKG_HOMEPAGE=https://sftpgo.com/
+TERMUX_PKG_DESCRIPTION="Full-featured and highly configurable SFTP, HTTP/S, FTP/S and WebDAV server"
+TERMUX_PKG_LICENSE="AGPL-V3"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="2.6.2"
+TERMUX_PKG_SRCURL=https://github.com/drakkan/sftpgo/releases/download/v$TERMUX_PKG_VERSION/sftpgo_v${TERMUX_PKG_VERSION}_src_with_deps.tar.xz
+TERMUX_PKG_SHA256=c90260b7b2901438bbd476eee9fd389af5af24113088a50284b2d170631b52ee
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_UPDATE_TAG_TYPE="latest-release-tag"
+TERMUX_PKG_HOSTBUILD=true
+
+termux_extract_src_archive() {
+	local file="$TERMUX_PKG_CACHEDIR/$(basename "${TERMUX_PKG_SRCURL}")"
+	mkdir -p "$TERMUX_PKG_SRCDIR"
+	tar xf "$file" -C "$TERMUX_PKG_SRCDIR" --strip-components=0
+}
+
+termux_step_host_build() {
+	termux_setup_golang
+
+	cd "$TERMUX_PKG_SRCDIR"
+	go build -mod vendor -o sftpgo
+	mv sftpgo "$TERMUX_PKG_HOSTBUILD_DIR"/sftpgo
+}
+
+termux_step_make() {
+	termux_setup_golang
+
+	go build -mod vendor -o sftpgo
+}
+
+termux_step_make_install() {
+	install -Dm700 -t "$TERMUX_PREFIX"/bin sftpgo
+	install -Dm600 -t "$TERMUX_PREFIX"/etc/sftpgo sftpgo.json
+
+	mkdir -p "$TERMUX_PREFIX"/share/sftpgo/{templates,static,openapi}
+	cp -Rf "$TERMUX_PKG_SRCDIR"/templates/* "$TERMUX_PREFIX"/share/sftpgo/templates
+	cp -Rf "$TERMUX_PKG_SRCDIR"/static/* "$TERMUX_PREFIX"/share/sftpgo/static
+	cp -Rf "$TERMUX_PKG_SRCDIR"/openapi/* "$TERMUX_PREFIX"/share/sftpgo/openapi
+
+	mkdir -p "$TERMUX_PREFIX"/share/bash-completion/completions
+	"$TERMUX_PKG_HOSTBUILD_DIR"/sftpgo gen completion bash > \
+		"$TERMUX_PREFIX"/share/bash-completion/completions/sftpgo
```

Comment:
Are there no completions for `fish`, `zsh`(, `elvish`, `nu`)?

Reply:
I'll add them later.

Reply:
Fair enough.